link test mentions incorrect build tags to fix certain tests failing

Hi :)

I'm new to the project and was trying to run some tests.

Attempting to run `go test ./htlcswitch` returns:

```         
--- FAIL: TestChannelLinkBandwidthConsistency (0.00s)
    link_test.go:2173: htlcswitch tests must be run with '-tags debug
--- FAIL: TestChannelLinkTrimCircuitsNoCommit (0.00s)
    link_test.go:2907: htlcswitch tests must be run with '-tags debug
--- FAIL: TestChannelLinkCleanupSpuriousResponses (1.24s)
    link_test.go:5162: alice fwdpkg index 0 should not have ack
--- FAIL: TestSwitchDustForwarding (17.43s)
    switch_test.go:3388: 
        	Error Trace:	switch_test.go:3388
        	Error:      	Target error should be in err chain:
        	            	expected: "dust threshold exceeded"
        	            	in chain: 
        	Test:       	TestSwitchDustForwarding
FAIL
FAIL	github.com/lightningnetwork/lnd/htlcswitch	108.933s
FAIL
```
Following the recommended resolution:

```
go test ./htlcswitch -tags debug
# github.com/lightningnetwork/lnd/build
build/log.go:88:39: undefined: LogLevel
FAIL	github.com/lightningnetwork/lnd/htlcswitch [build failed]
```

I dug through the Makefile and realised I needed to add the 'dev' tag. `go test ./htlcswitch -tags dev` works.

I'm guessing this is just an outdated hint that needs updating. :)

--- a/htlcswitch/link_test.go
+++ b/htlcswitch/link_test.go
@@ -2170,7 +2170,7 @@ func updateState(batchTick chan time.Time, link *channelLink,
 // sleep in this test and the one below
 func TestChannelLinkBandwidthConsistency(t *testing.T) {
 	if !build.IsDevBuild() {
-		t.Fatalf("htlcswitch tests must be run with '-tags debug")
+		t.Fatalf("htlcswitch tests must be run with '-tags dev")
 	}
 	t.Parallel()
 
@@ -2904,7 +2904,7 @@ func TestChannelLinkTrimCircuitsPending(t *testing.T) {
 // circuits if the ADDs corresponding to open circuits are never committed.
 func TestChannelLinkTrimCircuitsNoCommit(t *testing.T) {
 	if !build.IsDevBuild() {
-		t.Fatalf("htlcswitch tests must be run with '-tags debug")
+		t.Fatalf("htlcswitch tests must be run with '-tags dev")
 	}
 
 	t.Parallel()

